clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/deploy/ops/instances/TerminateDcosInstancesAtomicOperation.groovy

```diff
@@ -0,0 +1,75 @@
+  }
+
+  /*
+   * curl -X POST -H "Content-Type: application/json" -d '[ { "terminateInstances": { "appId": "dcos-test-v000", "hostId": "192.168.10.100", "account": "my-dcos-account" }} ]' localhost:7002/dcos/ops
+   * curl -X POST -H "Content-Type: application/json" -d '[ { "terminateInstances": { "appId": "dcos-test-v000", "hostId": "192.168.10.100", "wipe": true, "account": "my-dcos-account" }} ]' localhost:7002/dcos/ops
+   * curl -X POST -H "Content-Type: application/json" -d '[ { "terminateInstances": { "appId": "dcos-test-v000", "hostId": "192.168.10.100", "force": true, "account": "my-dcos-account" }} ]' localhost:7002/dcos/ops
+   * curl -X POST -H "Content-Type: application/json" -d '[ { "terminateInstances": { "appId": "dcos-test-v000", "hostId": "192.168.10.100", "wipe": true, "force": true, "account": "my-dcos-account" }} ]' localhost:7002/dcos/ops
+   * curl -X POST -H "Content-Type: application/json" -d '[ { "terminateInstances": { "appId": "dcos-test-v000", "taskIds": ["dcos-test-v000.asdfkjashdfkjashd"], "account": "my-dcos-account" }} ]' localhost:7002/dcos/ops
+   * curl -X POST -H "Content-Type: application/json" -d '[ { "terminateInstances": { "appId": "dcos-test-v000", "taskIds": ["dcos-test-v000.asdfkjashdfkjashd"], "wipe": true, "account": "my-dcos-account" }} ]' localhost:7002/dcos/ops
+   * curl -X POST -H "Content-Type: application/json" -d '[ { "terminateInstances": { "appId": "dcos-test-v000", "taskIds": ["dcos-test-v000.asdfkjashdfkjashd"], "force": true, "account": "my-dcos-account" }} ]' localhost:7002/dcos/ops
+   * curl -X POST -H "Content-Type: application/json" -d '[ { "terminateInstances": { "appId": "dcos-test-v000", "taskIds": ["dcos-test-v000.asdfkjashdfkjashd"], "wipe": true, "force": true, "account": "my-dcos-account" }} ]' localhost:7002/dcos/ops
+   * curl -X POST -H "Content-Type: application/json" -d '[ { "terminateInstances": { "taskIds": ["dcos-test-v000.asdf1", "dcos-test-v000.asdf2"], "account": "my-dcos-account" }} ]' localhost:7002/dcos/ops
+   * curl -X POST -H "Content-Type: application/json" -d '[ { "terminateInstances": { "taskIds": ["dcos-test-v000.asdf1", "dcos-test-v000.asdf2"], "wipe": true, "account": "my-dcos-account" }} ]' localhost:7002/dcos/ops
+   * curl -X POST -H "Content-Type: application/json" -d '[ { "terminateInstances": { "taskIds": ["dcos-test-v000.asdf1", "dcos-test-v000.asdf2"], "force": true, "account": "my-dcos-account" }} ]' localhost:7002/dcos/ops
+   * curl -X POST -H "Content-Type: application/json" -d '[ { "terminateInstances": { "taskIds": ["dcos-test-v000.asdf1", "dcos-test-v000.asdf2"], "wipe": true, "force": true, "account": "my-dcos-account" }} ]' localhost:7002/dcos/ops
+   */
+  @Override
+  Void operate(List priorOutputs) {
+    task.updateStatus BASE_PHASE, "Initializing termination of instances..."
+
+    def dcosClient = dcosClientProvider.getDcosClient(description.credentials)
+
+    if (description.appId) {
+      if (description.hostId) {
```

Comment:
Should we log something or throw an exception if somehow there's an appId but no hostId or taskIds?